Makefile

```diff
@@ -54,7 +54,6 @@ __build-gator:
 .PHONY: generate
 generate: __build-gomplate
 	$(docker) run \
-		-u $(shell id -u):$(shell id -g) \
```

Comment:
Was this causing an issue?

Reply:
Sorry, accidental commit. Yes it causes problems when running with rootless podman.

https://github.com/open-policy-agent/gatekeeper/pull/2632#pullrequestreview-1349305142